.github/actions/kibana-ftr/action.yml

```diff
@@ -0,0 +1,79 @@
+name: 'Run Kibana FTR'
+description: 'Run Kibana tests using FTR'
+
+inputs:
+  test_kibana_url:
+    description: 'URL for the Kibana instance to test'
+    required: true
+  test_es_url:
+    description: 'URL for the ES instance'
+    required: true
+  es_version:
```

Comment:
i think this should just be the kibana version we're running, as kibana/elasticsearch version should match. the kibana version is available at `package.json#version`. maybe add a checkbox instead to indicate if this is a pre-released version

Reply:
The `create-environment` workflow may deploy three types of versions: currently in development, pre-release, and released versions. According to the [documentation (3. Custom option)](https://www.elastic.co/guide/en/kibana/current/development-tests.html#_running_functional_tests), _you must run the same branch of tests as the version of Kibana you’re testing. To run against a previous minor version, use the --es-version <instance version> option._ Therefore, we need to pass the version string explicitly. If we replace this with a flag, additional logic would be required in the kibana-ftr action to calculate the version. Since the create-environment workflow already knows this version and provides it as an input, I believe it's more efficient to pass the version string directly rather than using a flag.